Description: Upstream changes introduced in version 2010.05-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 openscadpy (2010.05-1) natty; urgency=low
 .
   * Initial packaging. Inspired by http://gitorious.org/openscad-debian/openscad
 .
 The person named in the Author field signed this changelog entry.
Author: Henning Meyer <hmeyer@gmx.eu>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ openscadpy-2010.05/openscad.kdev4
@@ -0,0 +1,3 @@
+[Project]
+Manager=KDevCustomMakeManager
+Name=openscad
--- /dev/null
+++ openscadpy-2010.05/.kdev4/openscad.kdev4
@@ -0,0 +1,31 @@
+[Buildset]
+BuildItems=@Variant(\x00\x00\x00\t\x00\x00\x00\x00\x01\x00\x00\x00\x0b\x00\x00\x00\x00\x01\x00\x00\x00\x10\x00o\x00p\x00e\x00n\x00s\x00c\x00a\x00d)
+
+[Launch]
+Launch Configurations=Launch Configuration 0
+
+[Launch][Launch Configuration 0]
+Configured Launch Modes=execute
+Configured Launchers=nativeAppLauncher
+Name=New Native Application Configuration
+Type=Native Application
+
+[Launch][Launch Configuration 0][Data]
+Arguments=
+Debugger Shell=
+Dependencies=@Variant(\x00\x00\x00\t\x00\x00\x00\x00\x00)
+Dependency Action=Nothing
+Display Demangle Names=true
+Display Static Members=false
+EnvironmentGroup=default
+Executable=file:///home/hmeyer/devel/openscad/openscad
+GDB Path=
+Remote GDB Config Script=
+Remote GDB Run Script=
+Remote GDB Shell Script=
+Try Setting Breakpoints On Loading Libraries=true
+Working Directory=
+isExecutable=true
+
+[MakeBuilder]
+Number Of Jobs=4
--- openscadpy-2010.05.orig/opencsg.pri
+++ openscadpy-2010.05/opencsg.pri
@@ -6,16 +6,15 @@ opencsg {
   HEADERS += src/render-opencsg.h
   SOURCES += src/render-opencsg.cc
 
-  isEmpty(DEPLOYDIR) {
-    # Optionally specify location of OpenCSG using the 
-    # OPENCSGDIR env. variable
-    OPENCSG_DIR = $$(OPENCSGDIR)
-    !isEmpty(OPENCSG_DIR) {
-      INCLUDEPATH += $$OPENCSG_DIR/include
-      LIBS += -L$$OPENCSG_DIR/lib
-      message("OpenCSG location: $$OPENCSG_DIR")
-    }
-  }
+  OPENCSG_DIR = opencsg/OpenCSG-1.3.1
+  INCLUDEPATH += $$OPENCSG_DIR/include
+  LIBS += -L$$OPENCSG_DIR/lib
+  PRE_TARGETDEPS += $$OPENCSG_DIR/lib/libopencsg.a
+  LIBS += -Wl,-Bstatic -lopencsg -Wl,-Bdynamic
 
-  LIBS += -lopencsg
+	opencsg.target = "$$OPENCSG_DIR/lib/libopencsg.a"
+	opencsg.commands = cd opencsg;$(MAKE)
+
+  QMAKE_EXTRA_TARGETS += opencsg
+  QMAKE_CLEAN += -r opencsg/OpenCSG-1.3.1 opencsg/OpenCSG-1.3.1.tar.gz
 }
--- /dev/null
+++ openscadpy-2010.05/opencsg/Makefile
@@ -0,0 +1,17 @@
+all: OpenCSG-1.3.1/lib/libopencsg.a
+
+clean:
+	rm -rf OpenCSG-1.3.1*
+
+OpenCSG-1.3.1/lib/libopencsg.a: OpenCSG-1.3.1/.modmak
+	cd OpenCSG-1.3.1/src;$(MAKE)
+
+OpenCSG-1.3.1/.modmak: OpenCSG-1.3.1
+	awk 'NR==137{$$0="all: Makefile ../lib/$$(TARGET) $$(TARGETA)"}1' OpenCSG-1.3.1/src/Makefile >t
+	mv t OpenCSG-1.3.1/src/Makefile
+
+OpenCSG-1.3.1: OpenCSG-1.3.1.tar.gz
+	tar xzf OpenCSG-1.3.1.tar.gz
+
+OpenCSG-1.3.1.tar.gz:
+	wget http://www.opencsg.org/OpenCSG-1.3.1.tar.gz
--- openscadpy-2010.05.orig/openscadpy.pro
+++ openscadpy-2010.05/openscadpy.pro
@@ -49,7 +49,7 @@ macx {
     CONFIG += x86 x86_64
   }
 
-  TARGET = OpenSCAD
+  TARGET = OpenSCADpy
   ICON = icons/OpenSCAD.icns
   QMAKE_INFO_PLIST = Info.plist
   APP_RESOURCES.path = Contents/Resources
